Translation refactoring

SebastianoPistore opened the pull request:
- header standardized for all files based on this model

```
# Translation of amule interface to English
# aMule i18n resource file.
# Copyright (c) 2003-2020 aMule Team
# This file is distributed under the same license as the aMule package.
#
# TranslatorName <ilove@amule.org>, 2020.
```


It is now easier to understand the content of the file, the language of the translation, the copyright holder, the license and the names of all the people who work / worked on a translation.

- PO aligned with the latest version of POT (not translated but ready to resume translation)

- updated translations IT and IT_CH

Vollstrecker commented:
Sorry, but that won't work. As we use po4a dir translation generation, your changes will BE overwritten on the next run.

As I'm not at home, I can't tell you from my mind how to do this, but the docs are out there.

SebastianoPistore commented:
> Sorry, but that won't work. As we use po4a dir translation generation, your changes will BE overwritten on the next run.
> 

Ok: thanks for telling me, so copying the updated po is not enough. I took a quick look at po4a and it looks very versatile, I didn't know it.
But I did not understand the workflow exactly: generate the pot from the program code, update the po from the pot and then? How do you put them on
translations in the program? They generate the mo file, right? And to prevent the subsequent update from overwriting what I did, how do you do it?
Thanks

Vollstrecker commented:
Ups, sorry my bad. po4a is used for the manpages. po-files already are translated as po-files.
Short question: were did you get the additional lines that are deleted? de.po has many deletions beginning from 7643, while our file ends at 7644.

SebastianoPistore commented:
> Short question: were did you get the additional lines that are deleted? de.po has many deletions > beginning from 7643, while our file ends at 7644.

I deleted lines due to the updating of the file: I updated de.po with the pot, then deleted the string (which for the most part are changed sentences).
For example:
"Copyright (C) 2003-2008 aMule Team \n"
was no longer used, while
"Copyright (c) 2003-2019 aMule Team \n"
has been added to amule.
So both were present in the file I deleted the first one, so the code is cleaner.

gonosztopi commented:
I squashed and committed this pull request as 2027b0c3e62a1b88d1b3f6329e96c79ffd6cd0dc, only GitHub doesn't recognize it as being merged, therefore closing now.
